src/Traits/MailCryptPrivateSecretTrait.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Traits;
+
+/**
+ * @author doobry <doobry@systemli.org>
+ */
+trait MailCryptPrivateSecretTrait
```

Comment:
I'm struggling with the name of this property. Can you please explain what PrivateSecret means?

Reply:
The distinction between key and secret is as follows: keys contains the plain key file. secret contains a salt, nonce and libsodium secret box, which in turn contains the encrypted private key. So `MailCryptPrivateSecret` means: secret containing the private key. Same with `RecoverySecret`.

Reply:
Than it should be `MailCryptSecret` to align with `RecoverySecret`, shouldn't it?

Reply:
Debatable. It only holds the private key, the public key is in a separate database field (and has its own trait `MailCryptPublicKeyTrait`). One could argue that it should be `MailCryptPrivateKeySecretTrait` as well. I'm also fine with `MailCryptSecret` if you think it's a better name?

Reply:
Renamed after internal discussion :)